internal/apptracker/sentry/sentry_tracker.go

```diff
@@ -0,0 +1,42 @@
+package sentry
+
+import (
+	"time"
+
+	"github.com/getsentry/sentry-go"
+)
+
+// We need these variables to be able to mock sentry.CaptureMessage and sentry.CaptureException in tests since
+// package level functions cannot be mocked
+var (
+	captureMessageFunc   = sentry.CaptureMessage
+	captureExceptionFunc = sentry.CaptureException
+	InitFunc             = sentry.Init
+	FlushFunc            = sentry.Flush
+	RecoverFunc          = sentry.Recover
+)
+
+type SentryTracker struct {
+	FlushFreq int64
```

Comment:
Why do we need this value, it seems it's not being used. Shall we remove it?

Reply:
that's a good point, where are we using it?

Reply:
good point, removed